src/Helpers/Columnar.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Firesphere\SolrSearch\Helpers;
+
+use SilverStripe\Dev\Debug;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\FieldType\DBField;
+use SilverStripe\ORM\SS_List;
+use SilverStripe\View\ArrayData;
+
+class Columnar
+{
+    protected $component;
+    protected $columns = [];
+    protected $columnName = '';
+
+    public function __construct($component, $columns = [])
+    {
+        $this->component = $component;
+        $this->columns = $columns;
+        $this->columnName = $this->columns ? array_shift($this->columns) : null;
+    }
+
+    public static function make($component, $columns)
```

Comment:
Are you sure you want to use statics here?

Reply:
For now, Yes, since this is wip, once I am at the refactor stage, will see if this sugar is not needed anymore.